orca-deploymentmonitor/src/main/java/com/netflix/spinnaker/orca/deploymentmonitor/models/RequestBase.java

```diff
@@ -30,6 +30,7 @@
   private String account;
   private String region;
   private String cloudProvider;
+  private boolean isFailure;
```

Comment:
i don't think this is needed in `RequestBase`, just in the `DeploymentCompletedRequest`